content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -71,6 +71,10 @@ An _output entry_ (`outs`) consists of these fields:
   _checksum_ for HDFS.
 - `cache`: Whether or not this file or directory is <abbr>cached</abbr> (`true`
   by default, if not present). See the `--no-commit` option of `dvc add`.
+- `backup`: Whether or not this file or directory is pushed and pulled to/from
+  [remote storage](/doc/command-reference/remote) (`true` by default for
```

Comment:
Wait... So what happens to a regular output after it has been pushed if you then set it to backup:false manually, and run `pull`? Do `pull/fetch` throw an error? What's the message?

Reply:
Good catch: there wasn't one, it was an unhandled error. Now, you get an error for `pull/fetch` if the file isn't already in your local cache:

```
ERROR: failed to pull data from the cloud - Checkout failed for following targets:
file.txt
Is your cache up to date?
```

Reply:
Nice, thanks. Will follow up in the core PR.